### PR TITLE
[Bug_Fix] [Stateless_Testing] Fixed issues with NW initiated detach not handled for stateless ics drop test cases

### PR DESCRIPTION
**Pull request by VinashakAnkitAman**
## Title
[Bug_Fix] [Stateless_Testing] Fixed issues with nw initiated detach not handled for stateless ics drop test cases

## Summary
The NW Initiated detach requests were getting dropped for test case "s1aptests/test_attach_ics_drop_with_mme_restart.py". The reason was that the ICS req dropped flag was not getting reset from test script and hence for any new DL request, the message was getting dropped. This PR fixes the issue.

## Test Plan
Verified with Sanity and individual test case executions

### CODE REVIEW

File: TestCntlrApp/src/enbApp/nb.c [hunk 1]
```diff
@@ -1102,20 +1102,41 @@ PUBLIC S16 NbEnbInitCtxtSetupFail(NbInitCtxtSetupFail *initCtxtSetupFail)
...
+PUBLIC S16 NbEnbDropInitCtxtSetup(NbDropInitCtxtSetup *dropInitCtxtSetup) {
+  NB_LOG_ENTERFN(&nbCb);
 
-   if(NULLP == dropInitCtxtSetup)
-   {
-      NB_LOG_ERROR(&nbCb, "Recieved empty(NULL) request");
-      RETVALUE(RFAILED);
-   }
+  if (NULLP == dropInitCtxtSetup) {
+    NB_LOG_ERROR(&nbCb, "Recieved empty(NULL) request");
```
Comment (ssanadhya) [inline TestCntlrApp/src/enbApp/nb.c:1109]:
```suggestion
    NB_LOG_ERROR(&nbCb, "Received empty(NULL) request");
```

File: TestCntlrApp/src/enbApp/nb_tmr.c [hunk 2]
```diff
@@ -226,109 +226,105 @@ U32                          delay
...
  *
  * @details
  *
  *     Function: nbStopTmr
  *
  *         Processing steps:
- *               This function based upon the timer event reterives relevant timerCb and 
- *               tries to determine whether timer is running or not.In case timer 
- *               was running, it is stopped .
+ *               This function based upon the timer event reterives relevant
```
Comment (ssanadhya) [inline TestCntlrApp/src/enbApp/nb_tmr.c:236]:
```suggestion
 *               This function based upon the timer event retrieves relevant
```

**Review by ssanadhya — APPROVED** [review]
Minor spell check comments

File: TestCntlrApp/src/enbApp/nb_tmr.c (hunk 2 above)
Comment (VinashakAnkitAman) [reply TestCntlrApp/src/enbApp/nb_tmr.c]: Done

**Comment by VinashakAnkitAman** [comment]
> Minor spell check comments

Fixed all the marked spell comments.

**Pull request merged**